FluentUI/src/main/java/com/microsoft/fluentui/persona/AvatarGroupView.kt

```diff
@@ -0,0 +1,149 @@
+        maxDisplayedAvatars = styledAttrs.getInt(R.styleable.AvatarGroupView_maxDisplayedAvatars, DEFAULT_AVATAR_ALLOWED)
+        styledAttrs.recycle()
+    }
+
+    private var avatarList: List<IAvatar> = ArrayList()
+
+    var maxDisplayedAvatars: Int = DEFAULT_AVATAR_ALLOWED
+        set(value) {
+            if (field == value)
+                return
+
+            field = value
+            updateView()
+        }
+
+    /**
+     * Defines the [AvatarGroupStyle] applied to the avatar.
+     */
+    var avatarGroupStyle: AvatarGroupStyle = DEFAULT_AVATAR_GROUP_STYLE
+        set(value) {
+            if (field == value)
+                return
+
+            field = value
+            updateView()
+        }
+
+    /**
+     * Defines the [AvatarSize] applied to the avatar's height and width.
+     */
+    var avatarSize: AvatarSize = AvatarView.DEFAULT_AVATAR_SIZE
+        set(value) {
+            if (field == value)
+                return
+
+            field = value
+            updateView()
+        }
+
+    fun setAvatars(avatarList: List<IAvatar>) {
+        this.avatarList = avatarList
+        updateView()
+    }
+
+    private fun updateView() {
+        removeAllViews()
+        for ((index, avatar) in avatarList.withIndex()) {
+            if (index >= maxDisplayedAvatars) {
+                if (avatarList.size > maxDisplayedAvatars) {
+                    addOverFlowView(avatarList.size - maxDisplayedAvatars)
+                }
+                return
+            }
+            val avatarView = AvatarView(context)
+            avatarView.setAvatar(avatar)
+            avatarView.avatarSize = avatarSize
+            avatarView.avatarStyle = AvatarStyle.CIRCLE
+            avatarView.id = View.generateViewId()
+            val layoutParams = LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+            when(avatarGroupStyle) {
+                AvatarGroupStyle.PILE -> {
+                    avatarView.avatarBorderStyle = AvatarBorderStyle.NO_BORDER
+                    layoutParams.marginStart = index * (avatarView.getViewSize() + getPileSpacing())
+                }
+                AvatarGroupStyle.STACK -> {
+                    avatarView.avatarBorderStyle = AvatarBorderStyle.RING
+                    layoutParams.marginStart = index * (avatarView.getViewSize()/2 + getStackSpacing())
+                }
+            }
+            avatarView.layoutParams = layoutParams
+            addView(avatarView)
+        }
+    }
+
+    private fun addOverFlowView(i: Int) {
+        val avatarView = AvatarView(context)
+        avatarView.name = i.toString()
+        avatarView.avatarBackgroundColor = ContextCompat.getColor(context, R.color.fluentui_avatar_overflow_background)
+        avatarView.avatarSize = avatarSize
+        avatarView.avatarIsOverFlow = true
+        avatarView.avatarStyle = AvatarStyle.CIRCLE
+        avatarView.id = View.generateViewId()
```

Comment:
In a use case if user wants to click on one of the persona to see the persona card, how can we get ids of individual personas? or is it by design that click won't be supported?